components/Cms/Element/CmsElementHtml.vue

```diff
@@ -0,0 +1,13 @@
+<script setup lang="ts">
+import type { CmsElementText } from '@shopware-pwa/composables-next';
+
+const props = defineProps<{
+    element: CmsElementText;
+}>();
+</script>
```

Comment:
```suggestion
const props = defineProps<{
    element: CmsElementText;
}>();

const config = useCmsElementConfig(props.element);
const xx = config.getConfigValue('content');
</script>
```

i think the shopware way is better here. you can then replace the content from the v-html with the defined variable (in this example xxx)